src/utils.ts

```diff
@@ -314,10 +317,21 @@ export function getAPIFileType(filePath: string): string {
 /**
  * Checks if the network is available. Gracefully exits if not.
  */
-export async function checkIfOnline() {
+export async function safeIsOnline() {
```

Comment:
Would rather combine `safeIsOnline` and `checkIfOnline`.
Have a param with `checkIfOnline` like `checkIfOnline(canThrowError = false)`.